packages/helpers/src/i18n-create-interpolate-element.js

```diff
@@ -0,0 +1,24 @@
+import { createInterpolateElement } from "@wordpress/element";
+import { sprintf } from "@wordpress/i18n";
+
+/**
+ * Create an interpolated element from a translated string and catches errors.
+ *
+ * @param {string} translatedString The translated string to be interpolated.
+ * @param {[string]} args The arguments to be interpolated into the translated string.
+ * @param {object} conversionMap The conversion map for the interpolated values.
+ *
+ * @returns {object} The interpolated element.
+ */
+export const i18nCreateInterpolateElement = ( translatedString, args, conversionMap ) => {
```

Comment:
I think the variable naming can be improved here.
Also, the needing to write `args` in array form is a bit meh?